data.tf

```diff
@@ -1,3 +1,67 @@
+data "aws_iam_policy_document" "monitor_iam_access_sns_topic_policy" {
+  provider = aws.audit
+
+  statement {
+    effect  = "Allow"
+    actions = ["SNS:Publish"]
+
+    principals {
+      type        = "Service"
+      identifiers = ["events.amazonaws.com"]
+    }
+
+    condition {
+      test     = "StringEquals"
+      variable = "aws:PrincipalOrgID"
+
+      values = [
+        data.aws_organizations_organization.default.id
+      ]
+    }
+
+    resources = [aws_sns_topic.monitor_iam_access.arn]
+  }
+}
+
+data "aws_iam_role" "monitor_iam_access_audit" {
+  for_each = toset([for identity in try(var.monitor_iam_access, []) : identity.name if identity.type == "AssumedRole" && identity.account == "audit"])
+  provider = aws.audit
+  name     = each.value
+}
+
+data "aws_iam_user" "monitor_iam_access_audit" {
+  for_each  = toset([for identity in try(var.monitor_iam_access, []) : identity.name if identity.type == "IAMUser" && identity.account == "audit"])
+  provider  = aws.audit
+  user_name = each.value
+}
+
+data "aws_iam_role" "monitor_iam_access_logging" {
+  for_each = toset([for identity in try(var.monitor_iam_access, []) : identity.name if identity.type == "AssumedRole" && identity.account == "logging"])
+  provider = aws.logging
+  name     = each.value
+}
+
+data "aws_iam_user" "monitor_iam_access_logging" {
+  for_each  = toset([for identity in try(var.monitor_iam_access, []) : identity.name if identity.type == "IAMUser" && identity.account == "logging"])
+  provider  = aws.logging
+  user_name = each.value
+}
+
+data "aws_iam_role" "monitor_iam_access_master" {
+  for_each = toset([for identity in try(var.monitor_iam_access, []) : identity.name if identity.type == "AssumedRole" && identity.account == "master"])
+  name     = each.value
+}
+
+data "aws_iam_user" "monitor_iam_access_master" {
+  for_each  = toset([for identity in try(var.monitor_iam_access, []) : identity.name if identity.type == "IAMUser" && identity.account == "master"])
+  user_name = each.value
+}
+
 data "aws_organizations_organization" "default" {}
 
 data "aws_region" "current" {}
+
+data "aws_sns_topic" "all_config_notifications" {
+  provider = aws.audit
+  name     = "aws-controltower-AllConfigNotifications"
```

Comment:
If we're going to capitalise certain letters shouldn't we do it the whole string, e.g. 

```suggestion
  name     = "AWS-ControlTower-AllConfigNotifications"
```

What do you think?

Reply:
This is how AWS creates the Control Tower SNS Topics...I also think it's a bit strange but I believe there isn't much we can do...

![Screenshot 2020-11-17 at 09 21 57](https://user-images.githubusercontent.com/7815661/99364556-6777de80-28b6-11eb-8963-159bf007a736.png)

Reply:
Ah fair enough, if it's inline with other resources then all is good 👍